sklearn/multioutput.py

```diff
@@ -240,6 +240,34 @@ def predict(self, X):
     def _more_tags(self):
         return {"multioutput_only": True}
 
+    def get_metadata_routing(self):
+        """Get metadata routing of this object.
+
+        Please check :ref:`User Guide <metadata_routing>` on how the routing
+        mechanism works.
+
+        Returns
+        -------
+        routing : MetadataRouter
+            A :class:`~utils.metadata_routing.MetadataRouter` encapsulating
+            routing information.
+        """
+        router = (
+            MetadataRouter(owner=self.__class__.__name__).add(
+                estimator=self.estimator,
+                method_mapping=MethodMapping()
+                .add(callee="partial_fit", caller="partial_fit")
+                .add(callee="fit", caller="fit"),
+            )
+            # the fit method already accepts everything, therefore we don't
+            # specify parameters
+            .warn_on(child="estimator", method="fit", params=None)
```

Comment:
Looking through the code the string "estimator" comes from `add(estimator=` above, where `estimator` looks like a keyword argument. This behavior is not obvious, but I do not see a better API